tools_hive/conf/vep_plugins_hive_config.txt

```diff
@@ -158,4 +158,11 @@
       "file=[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/ClinPred/ClinPred_hg38_sorted_tabbed.tsv.gz"
     ]
   },
+  Paralogues => {
+    "params" => [
+      "matches=[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/Paralogues/Paralogues.pm_homo_sapiens_113_ClinVar_20240819.tsv.gz",
```

Comment:
@nuno-agostinho Is this file already available?

Reply:
Hi @imransl,

yes, the Paralogues plugin data should have been handed over for e113 - 
```
$ ls /nfs/production/flicek/ensembl/variation/enseweb-data_tools/grch38/e113/vep/plugin_data/Paralogues/
Paralogues.pm_homo_sapiens_113_ClinVar_20240819.tsv.gz  Paralogues.pm_homo_sapiens_113_ClinVar_20240819.tsv.gz.tbi
```

Reply:
@nakib103 Okay I can see it there now, but it has been added well after we had finished the syncing. Please let us know when new files like these become available again to sync them to the live directories.

Reply:
@imransl, yeah it was supposed to go in for e114 but we then added a bit earlier. I think it got lost in the process to let web team know about the new data. Sorry about that!

Reply:
No problem. The directories have been synced now, so I will merge this PR.